cmd/eksctl-anywhere/cmd/constants.go

```diff
@@ -9,6 +9,7 @@ const (
 	unhealthyMachineTimeoutFlag = "unhealthy-machine-timeout"
 	nodeStartupTimeoutFlag      = "node-startup-timeout"
 	noTimeoutsFlag              = "no-timeouts"
+	maxUnhealthyFlag            = "max-unhealthy"
```

Comment:
Should we just let the users configure this through the spec and not expose it via a flag?

Reply:
Removed the flag.